_posts/_2021-10-05-Launch-Announcement-1-1.markdown

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+authors: 
+  - henkle
+  - ahopp
+comments: true
+title: "OpenSearch 1.1 is here!"
+category:
+- releases
+---
+
+To the OpenSearch community everywhere, 
```

Comment:
Why is this written as a letter? It breaks continuity.

Reply:
Should be resolved.